Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -977,8 +990,7 @@ def planeFit(points):
         primary_direction = np.abs(n).argmax()
         if n[primary_direction] < 0.0:
             n *= -1.0
-        p = ctr
-        return p, n
+        return ctr, n
```

Comment:
```suggestion
        p = ctr
        return p, n
```

I suggest to keep the existing convention (as it was originally introduced) where `p` denotes `position` and `n` denotes `normal`

Reply:
I will use `p` instead of `ctr`.